dockstore-webservice/src/main/java/io/dockstore/webservice/resources/EntryResource.java

```diff
@@ -381,6 +385,31 @@ public List<VersionVerifiedPlatform> getVerifiedPlatforms(@Parameter(hidden = tr
         return verifiedVersions;
     }
 
+    @GET
+    @Path("/{entryId}/syncing")
+    @UnitOfWork
+    @Operation(operationId = "isSyncing", description = "Is this entry being updated automatically when the source repository changes?", security = @SecurityRequirement(name = JWT_SECURITY_DEFINITION_NAME))
+    @ApiResponse(responseCode = HttpStatus.SC_OK + "", description = "False if the entry is conclusively not being updated, true otherwise.",
+        content = @Content(mediaType = MediaType.APPLICATION_JSON, schema = @Schema(implementation = Boolean.class)))
+    public boolean isSyncing(
+        @Parameter(hidden = true, name = "user")@Auth User user,
```

Comment:
In the next commit, I will add a space before the `@Auth`, because although it's not preceded by a space in some existing endpoints, it feels more correct with the space.